python/ray/experimental/sgd/pytorch/pytorch_runner.py

```diff
@@ -91,6 +111,21 @@ def _create_loss(self):
         if torch.cuda.is_available():
             self.criterion = self.criterion.cuda()
 
+    def _create_schedulers_if_available(self):
+        if not self.scheduler_creator:
```

Comment:
Why is this necessary? Can the user add the scheduler after initializing the `DistributedPyTorchRunner`? Probably worth a comment.

Reply:
```suggestion
        # Learning rate schedules are optional.
        if not self.scheduler_creator:
```